strings.js

```diff
@@ -0,0 +1,51 @@
+// 1. Uppercase the first character
+
+function ucFirst(str) {
+    str = str[0].toUpperCase() + str.slice(1);
```

Comment:
Also, make all the other letters lowercase. For example:
`ucFirst('sOmEtHiNg'); // Something`

Reply:
done